kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue

```diff
@@ -105,42 +106,15 @@
     },
     mixins: [commonLearnStrings, commonCoreStrings],
     props: {
-      title: {
-        type: String,
-        required: true,
-      },
-      subtitle: {
-        type: String,
-        default: null,
-      },
       thumbnail: {
         type: String,
         default: null,
       },
-      channelThumbnail: {
-        type: String,
-        default: null,
-      },
-      channelTitle: {
-        type: String,
-        default: null,
-      },
       kind: {
```

Comment:
Any reason not to read this from the `content` prop too?

Reply:
Not a real reason. Would require changing how kind is passed to some child components since it's currently using `v-bind`, but that is a quick fix.